test/python/test_rticonnextdds_data_access.py

```diff
@@ -319,3 +350,49 @@ def test_reset_sequence(self, test_output, test_input):
     # The other fields are unchanged:
     assert sample.get_number("my_point.x") == 3
     assert sample.get_number("my_point_sequence#") == 2
+
+  def test_access_native_dynamic_data(self, populated_input):
+    get_member_count = rti.rti.DDS_DynamicData_get_member_count
+    get_member_count.restype = ctypes.c_uint
+    get_member_count.argtypes = [ctypes.c_void_p]
+    count = get_member_count(populated_input[0].native)
+    assert count > 0
+
+  def test_input_performance(self, populated_input):
+    numIter = 1000
+    sample = populated_input[0]
+
+    start = time.time()
+    for i in range (1, numIter):
+      v = sample.get_number("my_long")
+    end = time.time()
+    get_number_duration = end - start
+
+    start = time.time()
+    for i in range (1, numIter):
+      v = sample["my_long"]
+    end = time.time()
+    get_item_duration = end - start
+
+    print("__getitem__ is {:2.2%} slower than get_number".format(
```

Comment:
Do we know why it is slower? How much slower is it roughly?

Reply:
I think setitem is clear; the overhead of calling `isinstance`; I'm not sure about getitem. I will profile it.